Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMObjectsMappingWizardPage.xaml.cs

```diff
@@ -55,25 +55,42 @@ public POMObjectsMappingWizardPage()
             InitializeComponent();                       
         }
 
+        int i = 0;
+
         private void ElementsListCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
         {
             ElementInfo EI = ((ObservableList<ElementInfo>)sender).Last();
 
-            mWizard.IWindowExplorerDriver.UpdateElementInfoFields(EI);
-            EI.Locators = mWizard.IWindowExplorerDriver.GetElementLocators(EI);
-            EI.Properties = mWizard.IWindowExplorerDriver.GetElementProperties(EI);
-            EI.ElementName = GetBestElementName(EI);
-            EI.WindowExplorer = mWizard.IWindowExplorerDriver;
+            if (i == 20)
+            {
 
+            }
 
-            if (mSelectedElementTypesList.Contains(EI.ElementTypeEnum))
+            try
             {
-                mWizard.POM.MappedUIElements.Add(EI);
+                mWizard.IWindowExplorerDriver.UpdateElementInfoFields(EI);
+                EI.Locators = mWizard.IWindowExplorerDriver.GetElementLocators(EI);
+                EI.Properties = mWizard.IWindowExplorerDriver.GetElementProperties(EI);
+                EI.ElementName = GetBestElementName(EI);
+                EI.WindowExplorer = mWizard.IWindowExplorerDriver;
+
+
+                if (mSelectedElementTypesList.Contains(EI.ElementTypeEnum))
+                {
+                    mWizard.POM.MappedUIElements.Add(EI);
+                }
+                else
+                {
+                    mWizard.POM.UnMappedUIElements.Add(EI);
+                }
             }
-            else
+            catch (Exception ex)
             {
-                mWizard.POM.UnMappedUIElements.Add(EI);
+
             }
```

Comment:
you should not have empty catch block, add Reporter.ToLog with Error/Warn into it